common/logger/src/struct_log.rs

```diff
@@ -38,7 +38,6 @@ static NOP: NopStructLog = NopStructLog {};
 static mut STRUCT_LOGGER: &'static dyn StructLogSink = &NOP;
 static STRUCT_LOGGER_STATE: AtomicUsize = AtomicUsize::new(UNINITIALIZED);
 const UNINITIALIZED: usize = 0;
```

Comment:
why not use an enum?

Reply:
Future opportunity :D

Reply:
Specifically because this is state that needs to fit inside an AtomicUsize, based on my reading of the code....

But yeah Greg and I are going to come up with a plan to cleanup lots of this over the coming weeks.

Reply:
Ah, right I believe this code is [shamelessly copied from Rust Log](https://github.com/rust-lang/log/blob/b77ccefd7e09fa8e6811b7b02e519e566de4837f/src/lib.rs#L300-L311)